db/migrate/20220707162303_create_post.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :test
```

Comment:
- [ ] It should be text not test. Kindly change it.
- [ ] Also type should be text not string. Please change it.